README.md

```diff
@@ -1,195 +1,105 @@
+      ██║                             ██║  ██║
+███████╗██╗      ███████╗██╗  ██╗     ██║  ██║
+██╔════╝██║      ██╔════╝██║  ██║     ██║  ██║
+███████╗██║█████╗███████╗███████║     ██║  ██║
+╚════██║██║╚════╝╚════██║██╔══██║     ██║  ██║
+███████║███████╗ ███████║██║  ██║     ██║  ██║
+╚══════╝╚══════╝ ╚══════╝╚═╝  ╚═╝     ██║  ╚═╝
+ ██╗  ██║                             ██║
+ ██║  ██████████████████████████████████║
+ ██║  ╚═════════════════════════════════╝
+ ██║
+ ██║  ████████████████████████████████████████████╗
+ ██║  ╚═══════════════════════════════════════════╝
+ ██║
+ ╚═╝
+```
 
-Either command will leave you with a binary ```target/release/sl-sh``` that will run the shell.
-The above docker command will produce a completely static binary while compiling with rust will
-be linked to you systems libc.  You can use the musl target with cargo to produce a static binary
-with an installed rust.
 
-## Examples
-Currently sparse but if you grep lisp/ and contrib/ for a form you want an example of you will probably find one.
+# Simple Lisp Shell (pronounced slush)
 
-## ~/.config/sl-sh/slshrc extensions
-The following section details functionality to extend and modify the behavior of
-the shell.
+![Rust](https://github.com/sl-sh-dev/sl-sh/workflows/Rust/badge.svg?branch=master)
 
-### Prompt/PS1
-The command prompt for the shell is customize-able via the `__prompt` function. The
-function takes no arguments and expects a string to be returned. The canonical
-prompt in sl-sh would simply be:
-```
-	(defn __prompt()
-		(str "$ "))
-```
-
-### Command processing
-sl-sh offers two "hooks" for intercepting commands being executed:
-`__completion_hook` and `__exec_hook`.
-- `__completion_hook` is used to aid in tab completions. It takes a varargs and
-expects a list of string to be returned. For convenience :path and :default
-are also allowable return values. :path represents the list of paths and
-:default is :path concatenated with all valid sl-sh forms.
-- `__exec_hook` is an intercept function for every command sent to sl-sh. The
-function takes a string and expects a string or list to be returned. The string
-returned will be evaluated as a sl-sh form and a list will just be evaluated.
-This is useful if you want to check all input and modify said input if it meets
-certain conditions. For instance, if one were to decide that inputting any
-valid path to sl-sh should result in the shell changing the current working
-directory to that directory:
-```
-	(defn change-dir-if-arg-is-dir (cmd)
-			(if (fs-dir? cmd)
-				(str "cd " cmd)
-				cmd))
-
-	(defn __exec_hook (cmd-to-execute)
-		(let ((args-list (str-split " " cmd-to-execute)))
-				(match (length args-list)
-					(1 (change-dir-if-arg-is-dir (first args-list)))
-					(nil cmd-to-execute))))
-```
+Simple Lisp SHell (sl-sh) is a lisp based shell written in Rust. It is not POSIX
+compliant and makes no effort to be. It runs on the Linux and MacOS platforms.
```

Comment:
Should work on any unix but it not regularly tested on others yet.  I may be setting up something to cover a BSD at some point though (a test running on a Pi at home for instance).